update.json

```diff
@@ -3,8 +3,8 @@
     "remote-settings-devtools@mozilla.com": {
       "updates": [
         {
-          "version": "1.5.0",
-          "update_link": "https://github.com/mozilla-extensions/remote-settings-devtools/releases/download/1.4.0/remote-settings-devtools-1.4.0.xpi"
```

Comment:
The previous state was not correct. Thanks!

Reply:
I noticed!! 😬